controller.go

```diff
@@ -236,35 +244,35 @@ func main() {
 
 	kubeAdapter, err = kubernetes.NewAdapter(kubeConfig, ingressClassFiltersList, awsAdapter.SecurityGroupID())
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
This should Fatal, because of restarting automatically and trying to get AWS credentials from cache again